site/styles.css

```diff
@@ -1,3 +1,110 @@
 h1 {
-    color: red;
+    font-size: 80px;
+    color: white;
+    font-weight: bolder;
```

Comment:
lets go with `font-weight: bold` here. `bolder` is actually a value you can use but it does something different than `bold`.